fix: Updated Composer Electron app menu

tonyanziano opened the pull request:
## Description

> Currently we do not set the application menu, so Electron uses the default menu which contains links to Electron documentation.

This PR addresses that and explicitly sets the menu so that it only contains info / functionality relevant to Composer and not Electron.

## Task Item

closes #2832 

## Screenshots

### Windows

**File**

![image](https://user-images.githubusercontent.com/3452012/80631448-93ab3b00-8a0a-11ea-8599-ad1a3030f605.png)

**Edit**

![image](https://user-images.githubusercontent.com/3452012/80631470-9e65d000-8a0a-11ea-89d3-96848a7d9e00.png)

**View**

![image](https://user-images.githubusercontent.com/3452012/80631516-afaedc80-8a0a-11ea-9e0c-2e18a6572876.png)

**Window**

![image](https://user-images.githubusercontent.com/3452012/80631586-cbb27e00-8a0a-11ea-9207-e7077a609eff.png)


**Help**

![image](https://user-images.githubusercontent.com/3452012/80631602-d40ab900-8a0a-11ea-91c7-239322695141.png)


### Linux

**File**

![image](https://user-images.githubusercontent.com/3452012/80632571-5051cc00-8a0c-11ea-985d-1401d55142bd.png)

**Edit**

![image](https://user-images.githubusercontent.com/3452012/80632656-6cee0400-8a0c-11ea-91d8-e684909a90ad.png)


**View**'

![image](https://user-images.githubusercontent.com/3452012/80632706-7d9e7a00-8a0c-11ea-8d02-2f910527d31b.png)


**Window**

![image](https://user-images.githubusercontent.com/3452012/80632732-87c07880-8a0c-11ea-9741-7ef5e995066f.png)


**Help**

![image](https://user-images.githubusercontent.com/3452012/80632753-9018b380-8a0c-11ea-875d-174d889af444.png)


### Mac

I don't have a Mac, but feel free to add screen shots if you test it out!

cwhitten commented:
Things look good except for the Electron menu item. See attached.

![Screen Shot 2020-04-29 at 11 36 28 AM](https://user-images.githubusercontent.com/1156704/80633612-e3d7cc80-8a0d-11ea-991c-62ae57ade4f5.png)

Can we clean this up?

Can we also add a link in the help menu jumping out to the Composer docs? https://docs.microsoft.com/en-us/composer/

tonyanziano commented:
Doh! Yeah, I'll fix that and add a new menu item.

tonyanziano commented:
Updated "Help" menu:

![image](https://user-images.githubusercontent.com/3452012/80635009-27333a80-8a10-11ea-9b0b-9fe2595f3e26.png)

Also fixed up the labels on the Mac "App" menu
